go.mod

```diff
@@ -6,10 +6,12 @@ require (
 	connectrpc.com/connect v1.11.1
 	connectrpc.com/otelconnect v0.5.0
 	github.com/bufbuild/protocompile v0.6.0
+	github.com/bufbuild/protovalidate-go v0.3.3-0.20231002212436-9023bd754f7f
```

Comment:
This is from `go get ...@main`, perhaps we should tag a release on `protovalidate-go` first?

Reply:
https://github.com/bufbuild/protovalidate-go/releases/tag/v0.3.3

Reply:
Thanks for releasing, resolved in [662bbbc](https://github.com/bufbuild/buf/pull/2473/commits/662bbbc55e6ac76b3a9144d80bbda692b3d5f2ad).